packages/@aws-cdk/aws-stepfunctions/test/condition.test.ts

```diff
@@ -28,13 +28,85 @@ describe('Condition Variables', () => {
       },
     );
   }),
-  test('Exercise a number of other conditions', () => {
+  test('Exercise string conditions', () => {
```

Comment:
ideally we do not modify existing tests. it's preferred to only add new tests to ensure we don't introduce a regression. can we preserve this test as it was and test our conditions through new tests cases only?